src/main/java/programminglife/gui/controller/GuiController.java

```
@@ -124,6 +131,42 @@ public void update(Observable o, Object arg) {
     }
 
     /**
+     * Read out the file which contains all the recently opened files.
+     */
+    private void initRecent() {
+        try {
+            Files.createFile(new File("Recent.txt").toPath());
+        } catch (FileAlreadyExistsException e) {
+        } catch (IOException e) {
```

Please add a line here explaining that (and why) we ignore this exception